README.md

```diff
@@ -161,6 +161,12 @@ Type: `Boolean`, optional
 
 If you would like to pause or unpause the focus trap (see [`focus-trap`'s documentation](https://github.com/focus-trap/focus-trap#focustrappause)), toggle this prop.
 
+#### containerElements
+
+Type: `Array of Elements`, optional
```

Comment:
Hmm... I hadn't done my due diligence on this part yet.  Now that I'm looking better at it, here's the deal.

focus-trap-react is using `React.findDOMNode`, before and after my PR.  This has to take either a React element or an HTML element.  It can't take a string.

So, focus-trap-react doesn't pass down strings in one form or another to focus-trap.  It does elements.

I've updated the PropTypes and this section to only allow HTML elements, which is only partially true.  You _can_ pass in React elements, but I would avoid trying to advertise that, as using `findDOMNode` would be a  possible thing we could get rid of in this repo, as it's an escape hatch (https://reactjs.org/docs/react-dom.html#finddomnode).

I'm 50/50 on how to message it.  If you want me to include the fact that it could be a React element for containerElements, I can do that.  I'm just trying to not advertise that.

Thoughts?

I've pushed up my initial thought anyway.

Reply:
Good point. It would be nice to move away from `findDOMNode()` at some point, if possible. For now, though, let's not rock that boat. 🙂  But it does mean we have to be careful. The docs for ``findDOMNode()` say that if given a string, it'll return a text node with the string inside of it -- I don't think that will lead to expected behavior.

So with this new `containerElements` prop, we basically have two ways to specify trap containers: The prop, or children. The prop takes precedent, that makes sense. For `children`, it should be continue to be typed as [React.ReactNode](https://stackoverflow.com/a/58123882/6465363). It could be anything, but we can let React deal with that as it has been up until now.

The new `containerElements` prop, however, should probably strictly accept `HTMLElement` objects. I'd even go so far as to say that if we're processing an array of supposed elements from this prop (not from `children`), the component should throw an error if it finds anything in the array that isn't either falsy (undefined, null, false, empty string -- i.e. something we can easily filter out and ignore) or an `HTMLElement`.

`focus-trap` can handle strings as selectors, but because we put this through `findDOMNode()`, strings should be avoided in `containerElements` because of the behavior of that function WRT strings...

Thoughts?

Reply:
I agree that containerElements should just be HTML elements.  That is the prop type it is set up to be, and what the docs say it should be.

I started to add something to throw an error if what was passed in was both not falsey and not an HTML element, and then I thought to myself: isn't this exactly what the PropType is doing here?  Won't the proptype give warnings if they are using it incorrectly (obviously limited only to DEV mode, but still)?

So, we've told them what to pass, we show that in the docs, and they'll get warnings if they don't do it right.  Is there anything additionally that we should add to provide the checks you want?

Reply:
I'll do what you think.  I just think that it's probably already clear enough.  But, if you have a specific suggestion, I'd be happy to put it up.

Reply:
OK, see my [update here](https://github.com/focus-trap/focus-trap-react/pull/179#pullrequestreview-532580858).